web/html/citation-elements/blockquotes.md

```diff
@@ -21,7 +21,8 @@ aspects:
 
 
 links:
-  - '[MDN docs for blockquotes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote){website}'
+  - '[MDN docs for blockquotes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote){documentation}'
```

Comment:
can we have the resource title be the same/similar to the title found on the website? also, there is no need to say MDN docs, just add the {documentation} tags.

```suggestion
  - '[HTML <blockquote> Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote){documentation}'
```